### PR TITLE
[WIP] feat(release): apple

### PR DESCRIPTION
**Pull request by daniellacosse**
Currently refactoring the Fastfile so it uses both Development and Appstore certs. Need to finish that, then test everything.

### CODE REVIEW

**Review by fortuna — COMMENTED** [review]
Moving apple/ into src/ should be its own PR. Don't do it here.
Testing iOS and macOS in the same machine sounds good to me.

**Comment by daniellacosse** [comment]
> Moving apple/ into src/ should be its own PR. Don't do it here. Testing iOS and macOS in the same machine sounds good to me.

Good, that means cordova doesn't require it to be in the root

File: .github/workflows/build_and_test_debug.yml [hunk 1]
```diff
@@ -9,6 +9,10 @@ on:
     types:
       - opened
       - synchronize
+  pull_request_target:
```
Comment (fortuna) [inline .github/workflows/build_and_test_debug.yml:12]: I thought you meant we own't need pull_request_target with this PR

**Review by fortuna — COMMENTED** [review]
More info on signing for development: https://help.apple.com/xcode/mac/current/#/dev60b6fbbc7

File: .github/workflows/build_and_test_debug.yml (hunk 1 above)
Comment (daniellacosse) [reply .github/workflows/build_and_test_debug.yml:12]: ah, this is an outdated - if we put the dev certs in the repo like we discussed, this goes away

**Comment by daniellacosse** [comment]
closing in light of our meeting today

**Pull request closed without merging**